### ISSUE TITLE
Training loss increases by steps? Is that normal?

### ISSUE THREAD

**Issue by JingyuYang1997** [edgarschnfld/CADA-VAE-PyTorch#22]
Hi, thanks for your generous code contribution. I have a question about the training process. With AWA2 dataset, I observed that in the first 20 epoches,  the loss decreased. However later it will increase gradually by a large margin. Is that normal?
